definitions/scenarios/backup.rb

```diff
@@ -14,6 +14,14 @@ class Backup < ForemanMaintain::Scenario
       param :proxy_features, 'List of proxy features to backup (default: all)', :array => true
       param :skip_pulp_content, 'Skip Pulp content during backup'
       param :tar_volume_size, 'Size of tar volume (indicates splitting)'
+
+      preparation_steps do
+        Checks::Backup::IncrementalParentType.new(
```

Comment:
Could this just be the first step in the compose method?

Reply:
that's what I had before (and the current one doesn't work anyway), but it feels odd that it shows "preparation steps", those succeed, then it foes "now doing backup" and the first step fails the sanity check